src/docs/cookbook/design/package-fonts.md

```diff
@@ -157,3 +157,5 @@ class MyHomePage extends StatelessWidget {
 
 ![Package Fonts Demo](/images/cookbook/package-fonts.png){:.site-mobile-screenshot}
 
+[Pub website]: {{site.pub}}
```

Comment:
Pub website -> Pub site

This caused "][]" in the rendered page (https://sz-flutter.firebaseapp.com/docs/cookbook/design/package-fonts). Maybe you could search for that in your local copy of the generated static files?